00-Getting_Started/start_minikube.sh

```diff
@@ -5,10 +5,14 @@ set -e
 # Download the executable and the ISO
 echo Downloading minikube components
 if [[ ! -f minikube ]]; then
-  wget https://eleven.dev.kinvolk.io/u/alban/cloud-native-bpf-workshop/minikube
+  URL=https://eleven.dev.kinvolk.io/u/alban/cloud-native-bpf-workshop/minikube
+  URL=https://2020-08-17-cloud-native-bpf-workshop-public.s3.eu-central-1.amazonaws.com/minikube
+  time wget $URL
```

Comment:
I'm guessing this was a test, will you remove the extra URL and time?

Reply:
Updated :)